docs(em): update order elastic metal doc (#3756)
scaleway/docs-content: bare-metal/elastic-metal/how-to/create-server.mdx

1. Go to **Elastic Metal** in the **Bare Metal** section of the [Scaleway console](https://console.scaleway.com) side menu. The Elastic Metal server creation page displays.
2. Click **Create Elastic Metal server**. The server creation wizard will launch.
3. Complete the server creation wizard:
    - Select the geographical region for your server by choosing an Availability Zone.
      <Message type="note">
        Currently, Elastic Metal servers are available in:
        - Amsterdam, Netherlands (`AMS-1`, `AMS-2`)
        - Paris, France (`PAR-1`, `PAR-2`)
        - Warsaw, Poland (`WAW-2`, `WAW-3`)
      </Message>

>     - Choose the billing method. Available are either **hourly** or **monthly** billing.

Reviewer: ```suggestion
    - Choose your preferred billing method: **hourly** or **monthly**.
```
